autogen/docs/behaviorspace.md.mustache

```diff
@@ -138,10 +157,37 @@ a range.
 
 Also note that the double quotes around the variable names are required.
 
-**Subexperiment syntax:**For more advanced users, there is a third available syntax
-for varying parameters, the subexperiment syntax. This syntax allows you to define
-parameter combinations that are run separately, as opposed to being merged
-combinatorically. For example, if you wanted to try two values for a variable `a`
+All combinations of the specified values will be run. For example, if you have two
+ values for a variable `a` and three values of a variable `b` six runs will result.
+
+> `[["a" 1 2]`
+> `["b" [2 4 10]]`
```

Comment:
In the output once this is converted to HTML, the `>` quote lines get merged so these and the enumerations below and the Subexperiment samples all wind up on one line.  I don't think that's what is intended, and it is hard to make sense of it that way.   Maybe a code block would work better?